### PR TITLE
Issue #4 - Added Elizabeth's blogging tips slide

### PR DESCRIPTION
**Pull request by EChesters**
- Created images folder

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,105 +1,120 @@
...
-						<li>What is Object Oriented programming?</li>
-						<li>Heroku 101</li>
-						<li>How to set DNS server to point to amazon s3 bucket?</li>
-						<li>How to center anything with CSS</li>
-						<li>Which Javascript framework should i choose?</li>
-						<li>Mobile first design and development</li>
-						<li>Bootstrap vs Foundation?</li>
-					</ul>
-				</section>
+        <!-- Elizabeth's -->
```
Comment (tanyapowell) [inline index.html:48]: Comment isn't necessary. Your heading is `Elizabeth's Blogging` so it's pretty obvious it's your section 😈

**Comment by EChesters** [comment]
What the slide looks like:
screen shot 2016-04-25 at 18 08 25

File: index.html [hunk 2]
```diff
@@ -1,107 +1,120 @@
...
-					<ol>
-						<li>Keep a backlog of ideas that you want to explore</li>
-						<li>Blog for yourself</li>
-						<li>Your point of view is always unique</li>
-						<li>Try never to make assertions that you haven't tested</li>
-						<li>No topic is too small or insignificant</li>
-					</ol>
-				</section>
+        <section>
+          <h1>Tips from <a href="#">Ben Nadel</a></h1>
```
Comment (tanyapowell) [inline index.html]: The URL is missing 😢

File: index.html [hunk 3]
```diff
@@ -1,107 +1,120 @@
...
+      document.getElementsByTagName( 'head' )[0].appendChild( link );
+    </script>
+  </head>
+  <body>
+    <div class="reveal">
+      <div class="slides">
+        <section>
+          <img src="images/logo.png" alt="logo" class="plain logo">
+          <h2>Blogging Evening</h2>
+          <h3>April 27th @ <a href="#">GoCardless</a></h3>
```
Comment (tanyapowell) [inline index.html]: The URL is missing 😢

**Comment by tanyapowell** [comment]
Think you need to `pull` and then `merge` the `gh-pages` branch with your branch one more time, because not all of the changes I merged yesterday evening are present in this PR. Some are though (which is weird)

**Comment by tanyapowell** [comment]
👍 merge this bad boy! 👯

**Pull request merged**